src/libcore/iter/traits/iterator.rs

```diff
@@ -777,9 +767,7 @@ pub trait Iterator {
     /// assert_eq!(iter.next(), None);
     /// ```
     ///
-    /// [`Option<T>`]: ../../std/option/enum.Option.html
-    /// [`Some`]: ../../std/option/enum.Option.html#variant.Some
-    /// [`None`]: ../../std/option/enum.Option.html#variant.None
+    /// [`Option<T>`]: Option
```

Comment:
This is https://github.com/rust-lang/rust/issues/62834. This is a good workaround in the meantime though.

Reply:
But why not just use `Option` rather than `Option<T>`?

Reply:
`Option<T>` seems to be clearer for beginners/first readers of Iterator.

Reply:
We could use `Option<T>` in the book, but I don't think beginners get here easily.